static/css/custom.css

```diff
@@ -370,6 +370,10 @@ a:focus {
   .tile-heading {
     font-size: 2.2rem;
   }
+
+  .blog-logo img {
+    height: 50px;
```

Comment:
@rdebeasi  - by setting this height here on mobile view the height remains and looks a bit big compared to the menu bar across from it. I am OK with this, but other people might not be? just a thought anyway.